packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Statements/AttributeStatement.cs

```diff
@@ -12,30 +11,63 @@ public sealed class AttributeStatement : MethodBodyStatement
     {
         public CSharpType Type { get; }
         public IReadOnlyList<ValueExpression> Arguments { get; }
+        public IReadOnlyDictionary<string, ValueExpression> PositionalArguments { get; }
 
-        public AttributeStatement(CSharpType type, IReadOnlyList<ValueExpression> arguments)
+        public AttributeStatement(CSharpType type, IReadOnlyList<ValueExpression> arguments, IReadOnlyDictionary<string, ValueExpression> positionalArguments)
```

Comment:
Dictionaries have an ordering issue.
Should we use something like `IReadOnlyList<KeyValuePair<string, ValueExpression>>` to emphasize the order here?

Reply:
I think that is a good alternative.

Reply:
changed to `IReadOnlyList<KeyValuePair<string, ValueExpression>>`.
I also wonder if it makes more sense if we create a type for this `KeyValuePair<string, ValueExpression>`